src/test/java/com/snowflake/kafka/connector/internal/TestUtils.java

```diff
@@ -96,6 +96,11 @@ private static void init()
     conf.put(Utils.SF_URL, getProfile().get(HOST).asText());
     conf.put(Utils.SF_WAREHOUSE, getProfile().get(WAREHOUSE).asText());
     conf.put(Utils.SF_PRIVATE_KEY, getProfile().get(PRIVATE_KEY).asText());
+
+    JsonNode passphrase = getProfile().get(PRIVATE_KEY_PASSPHRASE);
+    if (passphrase!= null){
+    conf.put(Utils.PRIVATE_KEY_PASSPHRASE, passphrase.asText());
+  }
```

Comment:
Why do we need to add this?

Reply:
The test was failing if an encrypted private key has been used in `profile.json`.

Reply:
Thanks a lot for the PR. I created a PR to run the test: https://github.com/snowflakedb/snowflake-kafka-connector/pull/213. This is passphrase modification is breaking the test so I'll remove that for now.

Reply:
Removed passphrase modification and updated ConnectorTest based on #213